docs/source/conf.py

```diff
@@ -27,7 +27,8 @@
 version = u''
 # The full version, including alpha/beta/rc tags
 release = u'4.1.1'
-
+# Prefix link to always point to
+extlinks = {'prefix_link': ('https://github.com/huggingface/transformers/blob/v4.1.1/%s', '')}
```

Comment:
I think what @LysandreJik had in mind here was to use the `release` variable so each time we generate the docs on a given commit, the links are prefixed for that version.

But at the same time, it would also mean the master docs would always point to the latest release which is also something we don't want. So I think this line should be here but commented out, so that we uncomment it on the commit that is used to generate the docs of a specific version (with the proper tag using the `release` variable). This way when we regularly build the master doc at each PR, the links point to master, but when we make a release, we can just uncomment to build the links to the version released.

Does that make sense to you?

Reply:
I have made changes and committed. Is this what you wanted ? 

https://github.com/huggingface/transformers/pull/9217/files#diff-008dcb3426febd767787b1521f1fe33086313b927ea37eaab86df5fa88a51698R31

Reply:
Yes, it's good! Thanks a lot! Let's wait for @LysandreJik to be back to merge this if you don't mind.

Reply:
Yeah @sgugger I don't mind. We can wait.

Reply:
In the meantime, there seems to be some styling problems so make sure to run `make style` on your branch, and one error in the doc building step (I've added a comment in marian.rst where I think it comes from).

Reply:
Yeah was checking those,thought of fixing them in the meantime. Thanks!

Reply:
Why does `make style ` fix a lot of files which I haven't changed at all also when I try to use `style_doc.py` I get an error as here https://app.circleci.com/pipelines/github/huggingface/transformers/17685/workflows/f1f56681-8976-402a-95bd-387bd066ccb7/jobs/141134 that 10 files shoud be restyled ?! Am I missing something here ?
![image](https://user-images.githubusercontent.com/22686557/103140175-9ed57880-4709-11eb-9208-0df7656464dd.png)

Reply:
If I just do black `docs/source/conf.py` also there are changes from single quote to double quote 
![image](https://user-images.githubusercontent.com/22686557/103140260-87e35600-470a-11eb-9d36-a4627a374f5f.png)